docs/guides/adapting-new-services.md

```diff
@@ -1,39 +1,44 @@
 # Adapting New Services to Harmony<!-- omit in toc -->
 
-Please reach out in #harmony-service-providers (EOSDIS Slack) for additional guidance on any adaptation needs, and especially with any feedback that can help us improve.
+If you're adapting a new service for Harmony, we recommend reaching out in the `#harmony-service-providers` channel on EOSDIS Slack for help and to share any feedback that can improve this process for others.
 
-### Note Regarding services.yml
-Harmony services that are defined in the CMR UAT environment are configured in a file under the `config`
-directory called `services-uat.yml`, while services defined in the CMR production environment are
-configured in a file called `services-prod.yml`. These files are referred to collectively as `services.yml`
-in this document.
+### Note on `services.yml`
+Harmony services are configured differently depending on the environment:
+- Services deployed in the CMR UAT environment are defined in `config/services-uat.yml`.
```

Comment:
Shouldn't this read "Harmony UAT environment" and "Harmony production environment"? I know it has "cmr" in the actual files, but the environments are Harmony environments.

Reply:
Fixed.